src/App.tsx

```diff
@@ -15,58 +15,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type Good = (typeof goods)[number];
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGoods] = useState<Good | null>('Jam');
+  const selectedHandler = (good: Good | null) => {
```

Comment:
```suggestion
  const selectedGoodHandler = (good: Good | null) => {
```
better to understand